packages/utils/displayBalance.ts

```diff
@@ -0,0 +1,19 @@
+import { parseUnits } from 'viem'
```

Comment:
Could you help check the bundled file from uikit and see if the tree shaking is working as expected? We should only bundle `parseUnits` function instead of the whole viem package

Reply:
btw `viem` is peer dependency of `@pancakeswap/utils` right now.

 So we need to either put `viem` as devDependency of `@pancakeswap/uikit` or we change `viem` from peer dependency to dependency of `@pancakeswap/utils`